documentation/src/main/asciidoc/topics/dependencies/sb_remote.adoc

```diff
@@ -0,0 +1,8 @@
+[source,xml,options="nowrap"]
+----
+<dependency>
+    <groupId>org.infinispan</groupId>
+    <artifactId>infinispan-spring-boot-starter-remote-1.0.4.redhat-00002</artifactId>
```

Comment:
This is the dependency for 1.5, should we add the dependency for 2.1 too I guess, since both starters are productized ?